target_chains/near/receiver/Cargo.lock

```diff
@@ -1346,18 +1423,18 @@ dependencies = [
 
 [[package]]
 name = "home"
-version = "0.5.9"
+version = "0.5.11"
```

Comment:
This upgrades a dependency to a version that doesn't build on the pinned toolchain version, so the CI fails. Either revert the dependency update or bump the toolchain version.